stable/rabbitmq/templates/certs.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.rabbitmq.ssl.enabled }}
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Use a TLS secret instead. No credentials or sensitive data should be stored at configmaps.

Reply:
Changed to secrets